### PR TITLE
Feature/slt 508

### PR DESCRIPTION
**Pull request by agnis-mateuss**
Adds immediate backup validation.

After the backup has been dumped. Creates a temporary db container, imports db dump there and executes drush status.

Testing:
1. Enable backups in your deployment silta.yml
2. Trigger the cronjob.
3. Backup container in cronjobs' pod shows import logs and "Drush bootstrap Successful"
4. DB dump exists in the bucket storage.

### CODE REVIEW

File: charts/drupal/templates/_helpers.tpl [hunk 1]
```diff
@@ -457,3 +465,39 @@ set -e
...
+    TIME_WAITING=$((TIME_WAITING+1))
+
+    if [ $TIME_WAITING -gt 20 ]; then
+      echo "Database connection timeout"
+      exit 1
+    fi
+  done
+
+  
+  mysql -uroot -p{{ .db_password }} drupal --protocol=tcp < /tmp/db.sql
```
Comment (Jancis) [inline charts/drupal/templates/_helpers.tpl:500]: I'd suggest using env variables instead
```
mysql -u$DB_USER -p$DB_PASS $DB_NAME -h $DB_HOST --protocol=tcp < /tmp/db.sql
```

File: charts/drupal/templates/backup-cron.yaml [hunk 2]
```diff
@@ -15,9 +17,13 @@ spec:
   suspend: false
   jobTemplate:
     spec:
+      activeDeadlineSeconds: 300
+      backoffLimit: 2
       template:
         spec:
           enableServiceLinks: false
+          #Lets us kill DB process (in mariadb container) from backup container
```
Comment (Jancis) [inline charts/drupal/templates/backup-cron.yaml:25]: Whitespace after # is needed

File: charts/drupal/templates/backup-cron.yaml [hunk 3]
```diff
@@ -28,17 +34,38 @@ spec:
...
-                {{ include "drupal.backup-command" . | nindent 16 }}
+                {{ include "drupal.backup-command" (merge $params . ) | nindent 16 }}
+                {{ include "mariadb.db-validation" (merge $params . ) | nindent 16 }}
+            securityContext:
+              capabilities:
+                #For shareProcessNamespace
+                add:
+                  - SYS_PTRACE
+          - name: mariadb
+            image: docker.io/bitnami/mariadb:10.2.32
```
Comment (Jancis) [inline charts/drupal/templates/backup-cron.yaml:45]: Is it possible to skip the patch version on the image tag?

File: charts/drupal/templates/_helpers.tpl (hunk 1 above)
Comment (agnis-mateuss) [reply charts/drupal/templates/_helpers.tpl]: Edited in next commit.

File: charts/drupal/templates/backup-cron.yaml (hunk 2 above)
Comment (agnis-mateuss) [reply charts/drupal/templates/backup-cron.yaml]: Added space.

File: charts/drupal/templates/backup-cron.yaml (hunk 3 above)
Comment (agnis-mateuss) [reply charts/drupal/templates/backup-cron.yaml]: Yes, mariadb image repo provides a tag  with major.minor version. 
Updated in next commit.

**Review by Jancis — APPROVED** [review]

**Pull request merged**